### ISSUE TITLE
Feature Requests for Vega Strength 

### ISSUE THREAD

**Issue by nishith1210** [VarunS2002/Python-NSE-Option-Chain-Analyzer#66]
Hi Bro,
 I wanted to suggest a feature addition to GitHub regarding Vega strength logic, as explained by option trader Vibhor Gupta.

Here's the logic in a nutshell: Vega of option strikes between 0.5 to 0.6 Delta strikes is observed at market opening (9:15 am), and the same is calculated again at 10 am and afterwards. Based on the differences in Vega, market direction can be gauged.

You can find reference tweets explaining this concept here: [Vibhor Gupta's Tweet](https://twitter.com/VibhoreGupta5/status/1771032007042027892?s=19)

Integrating this logic could potentially enhance market analysis for us.. Thank you for considering this suggestion.

Best regards,
Nishith Singh

**Comment by rohitdave** [comment]
Yes if Vibhore's logic on Vega and Theta could be included it would be a powerful tool since Greeks data is updated at more or less at the same frequency as LTP as against OI 3 minutes

**Comment by rohitdave** [comment]
Or even the Greeks data  part from https://iluvmarkets.github.io/oc/?sheet=NIFTY
